packages/file_selector/file_selector/README.md

```diff
@@ -27,11 +28,23 @@ or read/write access:
 ```
 depending on your use case.
 
-### Examples
+# Feature supported platform
+Features support by platform:
+
+| Feature              | iOS      | Linux     | macOS     | Windows     | Android  |
```

Comment:
There's no need to list Android here since we already indicate at the top of the README (and pub will indicate as well) that Android is not supported.